internal/provider/provider.go

```diff
@@ -85,27 +88,27 @@ func (p *NuoDbaasProvider) Metadata(ctx context.Context, req provider.MetadataRe
 
 func (p *NuoDbaasProvider) Schema(ctx context.Context, req provider.SchemaRequest, resp *provider.SchemaResponse) {
 	resp.Schema = schema.Schema{
-		Description: "The NuoDB DBaaS provider provides the ability to manage the projects and databases running under the NuoDB Control Plane.",
+		Description: "The NuoDB DBaaS Provider manages NuoDB databases using the NuoDB Control Plane.",
```

Comment:
Why drop "projects" from the description? They are an implementation detail for most users, but it is still nice to have it noted.

Reply:
This is just a one sentence description of what the provider does, and I wanted to be concise. We will add resources for backups and backup policies as well eventually. We don't need to list every resource in this short description.